demo/style.css

```diff
@@ -122,13 +116,13 @@ aside ul a.current:before {
 }
 
 main {
-  margin-left: 18em;
+  margin-left: 19em;
   min-width: 36em;
-  padding: 1px 4em;
+  padding: 1.5em 3em 0;
```

Comment:
Or this is possibly the problem... hard to say.

Reply:
I don't think the two top titles are equatable in any way so they do not need to "line up" if that is what you were going for.